Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM ubuntu:24.04 AS exporter
+FROM ubuntu:24.04
 RUN apt-get update && \
     apt-get install python3-pip -y && \
+    rm -rf /var/lib/apt/lists/* && \
```

Comment:
Seems https://github.com/hadolint/hadolint/wiki/DL3009 will be a bit better.

Reply:
there is no need in our case